### PR TITLE
edit and clear testing

### PR DESCRIPTION
**Pull request by Ney2**
This is a PR for testing the edit and remove functionality of the to-do list.
1. Update the newTask.test.js file with the new test cases for editing and clearing all the completed tasks from the list.

### CODE REVIEW

File: src/Tests/newTask.test.js [hunk 1]
```diff
@@ -1,7 +1,3 @@
-/**
- * @jest-environment jsdom
- */
-
 import List from '../app/modules/class.js';
```
Comment (Ismailco) [inline src/Tests/newTask.test.js:1]: - [ ] You don't test any of the functions in your app, kindly create tests for the functions not just .pop() and .push(), these are just array methods. in this task, you need to test the functionality of the app to make sure they work properly

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @Ney2, @eudondian, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ney2** [comment]
1. Update display.test.js file 
2. Update files inside the modules folder (class.js, display.js, and newTask.js)

File: src/Tests/display.test.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    const listContainer = document.querySelectorAll('.listContainer li');
+    expect(listContainer).toHaveLength(3);
+  });
+});
+//   test('add task to the list', () => {
+//     list = ('submit my assignment', false, 3);
+//     todoList.push(list);
+//     expect(todoList).toHaveLength(3);
+//   });
+// });
```
Comment (AnselemOdims) [inline src/Tests/display.test.js:34]: - [ ] [OPTIONAL] Please kindly remove any code comments as it pollutes the codebase.

**Review by AnselemOdims — APPROVED** [review]
### Hi  @Ney2 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Tests/display.test.js (hunk 2 above)
Comment (Ney2) [reply src/Tests/display.test.js]: Thank you for your feedback. I noticed it after I submit but I removed it now.

File: src/Tests/newTask.test.js (hunk 1 above)
Comment (Ney2) [reply src/Tests/newTask.test.js:1]: Okay, I'll correct it. Thank you for your suggestion.

**Pull request merged**